easybuild/easyconfigs/m/multitail/multitail-6.2.1-intel-2015a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'multitail'
+version = '6.2.1'
+
+homepage = 'http://www.vanheusden.com/multitail/'
+description = """ MultiTail allows you to monitor logfiles and command output in multiple windows in a terminal, colorize, filter and merge."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [SOURCE_TGZ]
+source_urls = ['http://www.vanheusden.com/multitail/']
+
+patches = ['fix_makefile.patch']
```

Comment:
better version the patch filename: `multitail-6.2.1_eb.patch`, or atleast mention `multitail` in it

excluding the version might be OK here, it's unlikely that this will break easily for future releases (maybe just use `6`?)